javascript-loops-practice/countdown.js

```diff
@@ -0,0 +1,16 @@
+/* exported countdown */
+function countdown(number) {
+  if (number === 0) {
+    return [0];
+  } else {
+    var newNumber = countdown(number - 1);
```

Comment:
![image](https://user-images.githubusercontent.com/32880187/124200023-2de09600-da89-11eb-8d75-4713f281b6d7.png)

Reply:
sorry! just wanted to give myself a challenge! I changed it back to the while loop for this one.